astropy/units/core.py

```diff
@@ -2603,32 +2621,33 @@ def def_unit(
 
             {'latex': r'\\Omega'}
 
-    prefixes : bool or list, optional
+    prefixes : bool or iterable of UnitPrefix, optional
```

Comment:
This is needlessly complicated. My understanding is that passing in `True` is needed because the constant that contains the SI prefixes is not public, but it's not clear to me why that constant is private.